src/main/java/edu/wisc/my/ltiproxy/LTIParameters.java

```diff
@@ -0,0 +1,26 @@
+package edu.wisc.my.ltiproxy;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ *
+ * @author sibley
```

Comment:
I don't care for it either.  I've since edited the new file templates to omit this and license information.